webrtc.html

```diff
@@ -2504,6 +2504,31 @@
             </ol>
           </dd>
 
+          <dt>RTCRtpSender createRtpSender (DOMString kind)</dt>
+
+          <dd>
+            <p>Equivalent to
+            calling <a href="#dom-peerconnection-addtrack"><code>addTrack(null)</code></a>,
```

Comment:
Perhaps we should describe this differently since calling addTrack() with null results in a TypeError.

Should RTCRtpSender have a readonly type-attribute that is initialized when created (either with a MediaStreamTrack or a DOMString as described here)? That way you can read out the type if no track is set, and a new track can be verified against the predefined type.